src/main/java/de/dhbw/karlsruhe/model/Difficulty.java

```diff
@@ -0,0 +1,29 @@
+package de.dhbw.karlsruhe.model;
+
+import java.util.stream.Stream;
+
+public enum Difficulty {
+    EASY("e"),
+    MEDIUM("m"),
+    HARD("h");
+
+    public final String shortDifficultyName;
+
+    Difficulty(String shortDifficultyName) {
+        this.shortDifficultyName = shortDifficultyName;
+    }
+
+    @Override
+    public String toString() {
+        return this.name();
+    }
+
+    public static Stream<Difficulty> stream() {
```

Comment:
Brauchen wir die stream() Methode nur ein mal? Dann könnte man überlegen, ob man den Stream nicht einfach im DialogService Zeile 17 aufmacht. Das würde die statische Funktion hier einsparen. Was meinst du dazu?

Reply:
Momentan wird zweimal Difficulty.stream() in DifficultySelectionDialogService genutzt. Ich kenne die Best Practices zu so Streams nicht, kann mir aber vorstellen, dass beim Sudokugenerien auch der Stream genutzt werden könnte, falls weitere Eigenschaften zur Enum Difficulty dazugkommen. IDenkst du es ist dann sinnvoll es so zu lassen? Ich mache es dann, wie du meinst :)

Reply:
Best practice kenne ich in diesem Fall leider auch nicht :) Habe aber beides im Netz gefunden. Denke auf längere Sicht, ist es so gut wie du es gemacht hast.